pkg/cmd/cmd_test.go

```diff
@@ -0,0 +1,85 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"reflect"
+	"testing"
+	"io/ioutil"
+
+	"github.com/1dustindavis/gorilla/pkg/cmd"
```

Comment:
This import statement is invalid because `cmd_test.go` is part of the `cmd` package. A package can't import itself.

Reply:
removed invalid import in gorillashellout_test.go